.github/ISSUE_TEMPLATE/give_a_talk.yml

```diff
@@ -0,0 +1,43 @@
+name: Give a TTW talk 
+description: Resources and checklist for giving a talk about the Turing Way
+title: "[TALK] <title>"
+labels: [talks-and-workshops, newsletter]
```

Comment:
```suggestion
labels: [talks-and-workshops, newsletter]
assignees: [aleesteele]
```

If we want to ping Anne every time this issue is opened so she can add details to the calendar, the easiest thing to do would be to assign her to it. How do you feel about that, @aleesteele ?

Reply:
I'm okay with that! I need to get my github notification management system together, as I currently get pinged for everything in the repo. This means that I accidentally lose things amidst all the notifications. If I stop those global notifications, and only get pings for things I'm tagged in, this should be okay @sgibson91?

Reply:
This is really up to you, Anne! You will be receiving the "noise" so I would prefer to work this to fit _you_ rather than the other way around 💛

Reply:
@sgibson91 ❤️ A quick follow-up: I had a chat with @aldenc about this yesterday, and it might be a little tricky on the notification side of things. I currently get all alerts for the repository (that I can't turn off) because I'm an admin, which means that me specifically getting tagged in something might get lost in the wider list of notifications from the project. I think you are as well?

Thinking through alternatives: I wonder if there's a way for me to get pinged in a slack channel instead where github is integrated (i.e. everything tagged with 'newsletter' or 'talks-and-workshops' pings an alert into this channel? What do you think? I've been using the 'auto-github-updates' channel as a way of tracking the notifications in real time... I wonder if this could work?

Reply:
It's definitely not true that you have to receive all notifications because you are admin of a repo. I believe it is the default, but you can always change these settings via the "Watch" button at the top of the page. So I **don't** get all notifications even though I am an admin. The "Custom" setting might be interesting to look into...

<img width="445" alt="Screenshot 2022-08-16 at 16 44 02" src="https://user-images.githubusercontent.com/44771837/184923136-e7a32057-bc19-4ec1-b11a-666fc9545acf.png">

That aside, that doesn't mean setting up a Slack channel is a bad idea if that is what you would prefer to do. If you want to create a specific channel for these notifications or continue to use the "auto-github-updates" one I'm happy to configure notifications there

Reply:
Hmmm, the custom settings weren't as exciting as I'd hoped... (I was hoping for filtering by labels or created vs updated or something)

<img width="316" alt="Screenshot 2022-08-16 at 16 51 56" src="https://user-images.githubusercontent.com/44771837/184923838-19197850-c3a0-4c0f-a317-645bf441ecc5.png">

Let's go with the Slack channel